packages/dashboard/src/vaadin-dashboard-layout-mixin.js

```diff
@@ -85,8 +85,8 @@ export const DashboardLayoutMixin = (superClass) =>
           --_vaadin-dashboard-row-height: minmax(var(--_vaadin-dashboard-row-min-height, auto), auto);
 
           display: grid;
-          overflow: auto;
-          height: 100%;
+          overflow: hidden;
```

Comment:
Why `overflow: hidden` here? Wouldn't the default `visible` work fine? I know it’s unlikely, but perhaps some developer really needs very large shadows on the cards, and makes sure they are visible by adding padding on the host. If this internal element hides overflow, the shadows will be clipped. 

And now I realize that `--vaadin-dashboard-padding is on this element as well, not the host – why?

Reply:
Btw, I don't see the need to prefix the internal custom properties with `vaadin-dashboard` (e.g., `--_vaadin-dashboard-default-row-min-width` vs `--_default-row-min-width`). Just adds noise IMO.

Reply:
> Why `overflow: hidden` here? Wouldn't the default `visible` work fine? I know it’s unlikely, but perhaps some developer really needs very large shadows on the cards, and makes sure they are visible by adding padding on the host. If this internal element hides overflow, the shadows will be clipped.

It addresses a weird scroll/focus issue in editable dashboards on FF when padding and gap are set to zero. The padding property should be set to accommodate the shadow.

> And now I realize that `--vaadin-dashboard-padding is on this element as well, not the host – why?

Well, it _has_ to be there when we have `overflow:hidden` on it. I don't really see a problem with that?

Reply:
If `overflow: hidden` is required on the internal element because of Firefox, then the padding needs to be there as well, like you said.